### PR TITLE
Bugfix/working on windows

### PR DESCRIPTION
**Pull request by SalahDev86**
First code review

### CODE REVIEW

File: calculator/calculator.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>calculator</RootNamespace>
     <AssemblyName>calculator</AssemblyName>
-    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v7.0</TargetFrameworkVersion>
```
Comment (buamod) [inline calculator/calculator.csproj:11]: did you make the changes or Visual basic

File: calculator/Resources/layout/Main.axml [hunk 2]
```diff
@@ -4,15 +4,15 @@
     android:layout_width="fill_parent"
     android:layout_height="fill_parent">
     <EditText
-        android:id="@+id/editText2"
+        android:id="@+id/editText1"
```
Comment (buamod) [inline calculator/Resources/layout/Main.axml:7]: why this change?

File: calculator.sln [hunk 3]
```diff
@@ -1,6 +1,7 @@
-
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 2013
```
Comment (buamod) [inline calculator.sln:2]: I think my Visual Studio is 2012 and yours is 2013

File: calculator/Resources/layout/Main.axml (hunk 2 above)
Comment (SalahDev86) [reply calculator/Resources/layout/Main.axml:7]: to make the left edit text be the first in the calculation "e.g 4 - 2 is not 2 - 4" it was right to left now it is left to right

File: calculator.sln (hunk 3 above)
Comment (SalahDev86) [reply calculator.sln:2]: oh yes

File: calculator/calculator.csproj (hunk 1 above)
Comment (SalahDev86) [reply calculator/calculator.csproj:11]: i'm not sure what is this but i changed some settings

File: calculator/Resources/layout/Main.axml (hunk 2 above)
Comment (buamod) [reply calculator/Resources/layout/Main.axml:7]: ok

**Comment by buamod** [comment]
Ok so let me check this first on my machine then see if we can merge or not
